### DISCUSSION TITLE
tf are these lines for?

### DISCUSSION THREAD

**Discussion by libauror-a** [FunkinCrew/Funkin#1881] [Q&A] [answered]
https://github.com/ninjamuffin99/Funkin/blob/f94dece4a40d977c8a501b55ed9b9b3ee4344e13/source/PlayerSettings.hx#L8-L9

https://github.com/ninjamuffin99/Funkin/blob/f94dece4a40d977c8a501b55ed9b9b3ee4344e13/source/PlayerSettings.hx#L17-L23

https://github.com/ninjamuffin99/Funkin/blob/f94dece4a40d977c8a501b55ed9b9b3ee4344e13/source/PlayerSettings.hx#L33-L34

https://github.com/ninjamuffin99/Funkin/blob/f94dece4a40d977c8a501b55ed9b9b3ee4344e13/source/PlayerSettings.hx#L47-L117

what are these lines for? Avatars? Ningamuffin, did you do the thing of stealing code? :trollface:

**Comment by steviegt6** [comment, marked as answer]
Yep, looks like it came from various code used in Geokureli's advent2020 stuff: https://github.com/Geokureli/Advent2020/search?q=onAvatarAdd
TBF, credit was given and Geokureli has directly contributed many times.

**Reply by libauror-a** [reply]
cool thank
